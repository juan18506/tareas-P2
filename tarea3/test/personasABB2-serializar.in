crearArbolPersonas

# Construyo arbol 
crearPersona 7 7 Carlos
insertarPersonaArbolPersonas

crearPersona 20 20 Caroline
insertarPersonaArbolPersonas

crearPersona 2 2 Romina
insertarPersonaArbolPersonas

# imprimo arbol
imprimirArbolPersonas

# Transformo arbol en lista
serializarArbolPersonas

liberarArbolPersonas

Fin