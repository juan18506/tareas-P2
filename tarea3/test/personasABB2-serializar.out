1>2>#  Construyo arbol .
3>4>5>6>7>8>9>#  imprimo arbol.
10>Persona 2:  Romina, 2 años
Persona 7:  Carlos, 7 años
Persona 20:  Caroline, 20 años
11>#  Transformo arbol en lista.
12>Persona 7:  Carlos, 7 años
Persona 2:  Romina, 2 años
Persona 20:  Caroline, 20 años
13>14>Fin.
